src/platform/site-wide/announcements/components/PromoBanner.jsx

```diff
@@ -1,22 +1,7 @@
 import React from 'react';
 
 import PromoBanner from '@department-of-veterans-affairs/component-library/PromoBanner';
-import _recordEvent from 'platform/monitoring/record-event';
 
-export default function PromoBannerWithAnalytics({
-  recordEvent = _recordEvent,
-  onClose,
-  ...props
-}) {
-  return (
-    <PromoBanner
-      {...props}
-      onClose={() => {
-        recordEvent({
-          event: 'nav-promo-banner-link',
-        });
-        onClose();
-      }}
-    />
-  );
+export default function PromoBannerWithAnalytics({ ...props }) {
+  return <PromoBanner {...props} />;
```

Comment:
Can we just delete this whole component (now that it is abstracting nothing), and replaces instances of it with the `component-library/PromoBanner`?

FSYA that this probably can be removed too, `src/platform/site-wide/announcements/tests/components/PromoBanner.unit.spec.jsx`

Reply:
Yes indeed @ncksllvn.  Component and test have been removed.

Any suggestions on how to test this adequately when we have no current PromoBanner components in use?  I added one to my local Facility Locator and it seems to be working...

![image](https://user-images.githubusercontent.com/62304138/113329948-34c20600-92ec-11eb-9620-39378d0b8763.png)

Reply:
Sorry I missed this - but exactly how you did it is how I would've tested it too